Components/Queues/Drivers/AmqpQueueDriver.php

```diff
@@ -241,9 +243,14 @@ public function receiveNoWait()
         $start = microtime(true);
         $deserializationContext = new DeserializationContext();
 
-        /** @var QueueMessageInterface $msg */
-        $msg = $this->getSerializer()->deserialize($this->currentEnvelope->getBody(), SerializableInterface::class, $this->format, $deserializationContext);
-
+        try{
+            /** @var QueueMessageInterface $msg */
+            $msg = $this->getSerializer()->deserialize($this->currentEnvelope->getBody(), SerializableInterface::class, $this->format, $deserializationContext);
+        } catch (\Exception $exception) {
+            $this->getExceptionHandler()($exception, $this->currentEnvelope->getBody());
+            $this->ack();
+            return null;
```

Comment:
Shouldn't whatever is handling this exception be in charge of acking the message? Maybe we just want to log the error, tweak the message and continue processing it.